src/pool/srv_target.c

```diff
@@ -737,6 +737,36 @@ ds_pool_start(uuid_t uuid)
 	return rc;
 }
 
+/*
+ * Called via dss_thread_collective() to stop all container services
+ * on the current xstream.
+ */
+static int
+pool_child_stop_containers(void *uuid)
+{
+	struct ds_pool_child *child;
+
+	child = ds_pool_child_lookup(uuid);
+	if (child == NULL)
+		return 0;
+
+	ds_cont_child_stop_all(child);
+	ds_pool_child_put(child); /* -1 for the list */
```

Comment:
[Nit] Is the comment correct? It seems this put is for the lookup on line 749, while the put in ds_cont_child_stop_all is for the list. I think no comment is necessary here.

Reply:
sigh, stupid copy/paste error.